### ISSUE TITLE
Connection trough ftp by nitrado

### ISSUE THREAD

**Issue by Dannydaboy** [J0hnHawk/FS19_Webstats#57]
Hello,

I have try a lot of things but i can have a ftp connection on my server from nitrado is there a workaround?

**Comment by J0hnHawk** [comment]
Hi,
I dropped the FTP Support. We also had a dedicated server at nitrado. But as far as I know, Nitrado does not allow the access to the savegames by FTP.
So I created a new method to access the savegames. You'll need to add a user in the webinterface of the dedicated server. With this user data web stats can download the savegame.

If you want to test it by your own you can download the old version with ftp support: https://github.com/J0hnHawk/FS19_Webstats/tree/7de33b67d0f7596cffbc27cae43a2d812f7bea93

**Comment by Dannydaboy** [comment]
Hi its work pretty good,what is a different on this version and the newer version can i see a changelog?

**Comment by J0hnHawk** [comment]
There is no changelog so far. You need to scroll throug the commits.

**Comment by Dannydaboy** [comment]
It works great only the prices don’t work

**Comment by J0hnHawk** [comment]
How do you reach the savegame via FTP at nitrado? I get only to the mods if by login to the FTP server.
 
> It works great only the prices don’t work

Which map do you play on. If it is a modded map, you need to create a map config for it.

**Comment by Dannydaboy** [comment]
Yes it is a modded map what files must I modified for a mod map?

**Comment by J0hnHawk** [comment]
Copy the felsbrunn or ravenport folder to a new name. 
Then take a look to the items.xml of your savegame. Search for "SellingStationPlaceable" entrys. Then create an entry in your map config for each sellingstation. 

The sellstations for Felsbrunn are configured in the file tipTrigger.xml take a look into it and adapt it for your map config.

**Comment by ghost** [comment]
As I watch, it will probably be better to return the ftp option :)

**Comment by J0hnHawk** [comment]
> As I watch, it will probably be better to return the ftp option :)

Wouldn't be a problem but why?

**Comment by ghost** [comment]
Works for me, just a tip when I see that others are asking for it ...
